tasks/src/zero-shot-image-classification/about.md

```diff
@@ -8,4 +8,23 @@ The data in this learning paradigm consists of
 
 - Seen data - images and their corresponding labels
 - Unseen data -  only labels and no images
-- auxiliary information - additional information given to the model during training connecting the unseen and seen data. This can be in the form of textual description or word embeddings.
\ No newline at end of file
+- auxiliary information - additional information given to the model during training connecting the unseen and seen data. This can be in the form of textual description or word embeddings.
+
+
+## Use Cases
+
+### Autonomous Vehicles
+Autonomous Vehicles are a perfect use case for Zero-shot image classification. Occasionally in real-life situations, even we humans encounter new objects on the road that we are not sure how to react to instantly. Detecting + Classifying novel objects and deciding the correct actions to take are crucial for Robust Autonomous Vehicle Systems.
```

Comment:
I really don't think zero shot classification is used in something that requires a lot of safety. @sayakpaul do you know?

Reply:
Yeah, for safety applications, we wouldn't want to go zero-shot, in my opinion. In any safety-critical applications, it's better to have well-trained classifiers.

Reply:
Thanks alot for the insight here 😃